content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -31,46 +30,64 @@ options(stringsAsFactors = FALSE)
 ### What?
 
 A coordinate reference system (**CRS**) -- also called spatial reference system (SRS) -- is what you need if you want to interpret numeric coordinates as actual point locations with reference to the Earth.
-Two types of coordinate reference system exist: _geodetic_ and _projected_ CRSes.
+Two types of coordinate reference system exist: _geodetic_ and _projected_ CRSs.
 The former serve only to locate coordinates relative to a 3D model of the Earth surface, while the latter add a projection to generate coordinates on a 2D map.
-Coordinate operations convert or transform coordinates from one CRS to another, and you often need them because the CRS may differ between dataset 1, dataset 2 or a certain mapping technology.
+Coordinate operations convert or transform coordinates from one CRS to another, and you often need them because the CRS may differ between dataset 1, dataset 2 or a specific mapping technology.
```

Comment:
maybe add the example of leaflet, which wants the coordinates to be in epsg 4326 format

Reply:
OK!

Reply:
5e5ba2f3